### PR TITLE
Allow to set JAEGER_REPORTER_MAX_QUEUE_SIZE globally.

### PR DESCRIPTION
**Pull request by cyriltovena**
This also format some jsonnet.

Signed-off-by: Cyril Tovena <cyril.tovena@gmail.com>

### CODE REVIEW

File: production/ksonnet/loki/config.libsonnet [hunk 1]
```diff
@@ -40,6 +40,8 @@
     overrides_configmap_mount_name: 'overrides',
     overrides_configmap_mount_path: '/etc/loki/overrides',
 
+    jaeger_reporter_max_queue: 5000,
```
Comment (kavirajk) [inline production/ksonnet/loki/config.libsonnet:43]: Q: Curious what's the rationale behind this increase?

**Review by kavirajk — APPROVED** [review]
LGTM!. just one clarification!

File: production/ksonnet/loki/config.libsonnet (hunk 1 above)
Comment (cyriltovena) [reply production/ksonnet/loki/config.libsonnet:43]: Sorry this is from https://grafana.com/blog/2020/07/09/where-did-all-my-spans-go-a-guide-to-diagnosing-dropped-spans-in-jaeger-distributed-tracing/

Comment (cyriltovena) [reply production/ksonnet/loki/config.libsonnet:43]: Sorry there's no much context here, but the idea is thatI want to make it easier to set the queue size for tracing as we can see that Loki generally creates a lot of span that can be dropped.

May be the default is not so great though, I could tune down to 1k and do 5k internally for us.

Comment (cyriltovena) [reply production/ksonnet/loki/config.libsonnet:43]: @dannykopping wdyt ?

Comment (kavirajk) [reply production/ksonnet/loki/config.libsonnet:43]: Yea!. 5k may be too much for default!

**Review by dannykopping — APPROVED** [review]
LGTM :+1: 

It seems like your editor is formatting a lot of code which is unrelated to your change; if we can keep those unrelated changes out of future PRs it'd help with review (and also keep the git history clean).

**Comment by cyriltovena** [comment]
> LGTM 👍
> 
> It seems like your editor is formatting a lot of code which is unrelated to your change; if we can keep those unrelated changes out of future PRs it'd help with review (and also keep the git history clean).

It's actually `make fmt-jsonnet` in our make file that did it.

File: production/ksonnet/loki/config.libsonnet (hunk 1 above)
Comment (cyriltovena) [inline production/ksonnet/loki/config.libsonnet:43]:
```suggestion
    jaeger_reporter_max_queue: 1000,
```

**Pull request merged**